File: integration/feature/test-quick/resources/app/test/src/MyNumberCombinatorTests.scala
package app

import utest.*
import app.MyNumber

object MyNumberCombinatorTests extends TestSuite {
  def tests = Tests {
    test("simple") {
      val a = MyNumber(1)
      val b = MyNumber(2)
      val c = MyNumber(3)
      val result = MyNumber.combine(a, b, c)
      assert(result == MyNumber(6))
    }
  }
}

File: integration/feature/test-quick-java/resources/app/test/src/MyNumberCombinatorTests.java
package app;

import org.junit.Test;
import static org.junit.Assert.*;

public class MyNumberCombinatorTests {
    @Test
    public void simple() {
        MyNumber a = new MyNumber(1);
        MyNumber b = new MyNumber(2);
        MyNumber c = new MyNumber(3);
        MyNumber result = MyNumber.combine(a, b, c);
        assertEquals(new MyNumber(6), result);
    }
}
